cpp/controller/controller.cpp

```diff
@@ -160,27 +186,40 @@ void Controller::stop()
 
 void Controller::startBreak()
 {
-    timer().setElapsedBreakDuration(0);
-    timer().countBreakTime();
-
-    // update backup manager
-    m_backupManager.data().elapsedWorkTime = 0;
```

Comment:
Updating backup manager is removed. Is it expected?

Reply:
Yes, it is one in `Controller::onElapsedWorkTimeChange`.